edi-tools/modules/codegen/rest_connector_gen.bal

```diff
@@ -0,0 +1,36 @@
+
+function generateRESTConnector(string libName) returns string {
+    string restConCode = string `
+import ballerina/http;
+import ballerina/edi;
+
+configurable int restConnectorPort = 9090;
+
+service /${libName}EDI on new http:Listener(restConnectorPort) {
+
+    isolated resource function post reader/[string ediType](@http:Payload string ediData) returns json|error {
+        EDI_NAME|error ediTypeName = ediType.ensureType();
+        if ediTypeName is error {
+            return error edi:Error("Unsupported EDI type: " + ediType + ". " + ediTypeName.message());
```

Comment:
```suggestion
            return error edi:Error("Unsupported EDI type: " + ediType + "." + ediTypeName.message());
```

Reply:
Space is added there intentionally to make the error message readable.